Source/WebCore/platform/graphics/PathSegmentData.cpp

```diff
@@ -310,7 +310,7 @@ void PathArcTo::extendBoundingRect(const FloatPoint& currentPoint, const FloatPo
 {
     boundingRect.extend(currentPoint);
     boundingRect.extend(controlPoint1);
-    boundingRect.extend(calculateEndPoint(currentPoint));
+    boundingRect.extend(calculateArcToEndPoint(currentPoint, controlPoint1, controlPoint2, radius));
```

Comment:
I wonder if we could make this _ever so slightly_ more efficient by rewriting this to make it only call `maxX`/`maxY` and `setLocationAndSizeFromEdges` once.